tests/js/spec/components/featureTourModal.spec.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+
+import {mountWithTheme} from 'sentry-test/enzyme';
+
+import FeatureTourModal from 'app/components/modals/featureTourModal';
+import GlobalModal from 'app/components/globalModal';
+
+const steps = [
+  {
+    title: 'First',
+    body: 'First step',
+    image: <em data-test-id="step-image">Image</em>,
+    actions: (
+      <a href="#" data-test-id="step-action">
+        additional action
+      </a>
+    ),
+  },
+  {title: 'Second', body: 'Second step'},
+];
+
+describe('FeatureTourModal', function() {
+  let onAdvance, onCloseModal;
+
+  const createWrapper = () =>
+    mountWithTheme(
+      <React.Fragment>
+        <GlobalModal />
+        <FeatureTourModal steps={steps} onAdvance={onAdvance} onCloseModal={onCloseModal}>
```

Comment:
FeatureTourModal is a common function and not a valid JSX element... Please see:

![image](https://user-images.githubusercontent.com/29228205/90726190-9aafcf00-e2c1-11ea-9963-866b91f86fee.png)

Reply:
This should be fixed now. This error was caused by the directly returning `props.children()`. Typescript was getting confused about the types potentially because FeatureTourModal was a basic function and not a class component.

Reply:
Blerg it is fixed on another branch, I'll move those changes here.